### DISCUSSION TITLE
Failed to compile Cachebench

### DISCUSSION THREAD

**Discussion by Mi5sssss** [facebook/CacheLib#168] [Q&A]
When I used gcc to compile the main in cachebench, it went wrong like:
/cachelib/allocator/nvmcache/NvmCache-inl.h:254:16: error: missing template arguments before ‘b’
  254 |   folly::Baton b;
/cachelib/allocator/nvmcache/NvmCache-inl.h:266:9: error: ‘b’ was not declared in this scope
  266 |         b.post();
      |

**Comment by therealgymmy** [comment]
can you share repro instructions on your build failure? Also this seems more suitable as an issue instead of discussion.

**Comment by pbhandar2** [comment]
Closing this discussion as it has been inactive. If this is still relevant, then please create a new discussion post and we will get back to you as soon as we can. Thank you!
